### PR TITLE
uwc-docs: Updated User Guide for 1.6 release.

### PR DESCRIPTION
**Pull request by aditisa1**
Modified for UWC 1.6

Signed-off-by: aditi saxena <aditi.saxena@intel.com>

### CODE REVIEW

File: source/Pages/page_02.rst [hunk 1]
```diff
@@ -1,221 +1,58 @@
...
-    • Well defined message flow and payload to interact with the vendor application
-    • Policy-based execution of Soft-RT applications
-    • Supports multiple well pads and devices configurations
-    • Polls and controls multiple devices simultaneously
-    • Data Publish Subscribe ZeroMQ and MQTT
-    • Device Management – System Telemetry, over-the-network (a.k.a. over-the-air, or OTA) updates of firmware (FOTA), system software (SOTA), and applications (AOTA)
-    • Scalable-down to power-sensitive remote applications
-    • Scalable-up to edge analytics and vision through the EII base
+•	Enabled Data Persistence feature in UWC
+    • Integrated InfluxDBConnector & Telegraf services of EII's TICK stack timeseries services with UWC.
```
Comment (nagdeepgk) [inline source/Pages/page_02.rst:10]: In deployment https://open-edge-insights.github.io/uwc-docs/Pages/page_02.html, somehow these bullet points are not visible. Can we add bullet points or numericals ?

File: docs/_sources/Pages/page_04.rst.txt [hunk 2]
```diff
@@ -1,67 +1,130 @@
...
 
-            **from** *APT::Periodic::Update-Package-Lists* **"1"**
+    •	git clone https://github.com/open-edge-insights/eii-core.git IEdgeInsights --branch=v2.6
+    •	git clone https://github.com/open-edge-insights/eii-messagebus.git IEdgeInsights/common/libs/EIIMessageBus --branch=v2.6
+    •	git clone https://github.com/open-edge-insights/eii-c-utils.git IEdgeInsights/common/util/c --branch=v2.6
+    •  	git clone https://github.com/open-edge-insights/eii-zmq-broker.git IEdgeInsights/ZmqBroker --branch=v2.6
+    •	git clone https://github.com/open-edge-insights/ts-telegraf.git IEdgeInsights/Telegraf --branch=v2.6
+    •	git clone https://github.com/open-edge-insights/eii-influxdb-connector.git IEdgeInsights/InfluxDBConnector --branch=v2.6
+    •	git clone https://github.com/open-edge-insights/eii-etcd-ui.git IEdgeInsights/EtcdUI --branch=v2.6
+    •	git clone https://github.com/open-edge-insights/eii-tools.git IEdgeInsights/tools --branch=v2.6
```
Comment (nagdeepgk) [inline docs/_sources/Pages/page_04.rst.txt:27]: we also need to add 2 uwc repos with v1.6 tag, as mentioned. lets add them.

File: docs/_sources/Pages/page_04.rst.txt [hunk 3]
```diff
@@ -1,67 +1,130 @@
...
 
-    2.	Disable download upgradable packages by changing setting
+•	Please choose one of the below options based on Dev or Prod mode.
+    1) Dev
+    2) Prod 
+•	Please choose one of the below options based on the use case (combination of UWC services) needed.
+    1) Basic UWC micro-services without KPI-tactic Application & Sparkplug-Bridge - (Modbus-master TCP & RTU, mqtt-bridge, internal mqtt broker, ETCD server, ETCD UI & other base EII & UWC services)
+    2) Basic UWC micro-services as in option 1 along with KPI-tactic Application (Without Sparkplug-Bridge)
+    3) Basic UWC micro-services & KPI-tactic Application along with Sparkplug-Bridge
+    4) Basic UWC micro-services with Sparkplug-Bridge and no KPI-tactic Application
```
Comment (nagdeepgk) [inline docs/_sources/Pages/page_04.rst.txt:59]: there are only 4 use cases mentioned here. we need to update thi sfor our 8 use cases.

**Review by nagdeepgk — CHANGES_REQUESTED** [review]
@aditisa1 The data persistence changes from softdel's PR https://github.com/open-edge-insights/uwc-docs/pull/7, needs to be added to this, as mentioned earlier.
Request to plz add the changes.

**Comment by nagdeepgk** [comment]
@aditisa1 The data persistence changes from softdel's PR https://github.com/open-edge-insights/uwc-docs/pull/7 , needs to be added to this, as mentioned earlier.
Request to plz add the changes.

**Comment by aditisa1** [comment]
> @aditisa1 The data persistence changes from softdel's PR #7 , needs to be added to this, as mentioned earlier.
> Request to plz add the changes.

@nagdeepgk I made those changes I copied it . It may be possible that I moved the files to make page_02 (release notes) because which file structure is different. Did I miss anything?

File: source/Pages/page_02.rst (hunk 1 above)
Comment (aditisa1) [reply source/Pages/page_02.rst:10]: doc
vs

Comment (aditisa1) [reply source/Pages/page_02.rst:10]: Hi Nagdeep 
i see that is happening in all the file above is page_01.rst

File: source/Pages/page_02.rst [hunk 4]
```diff
@@ -1,221 +1,58 @@
...
 
-2.3.5  Configurations
-~~~~~~~~~~~~~~~~~~~~~
+------------------------------------
+2.4 Changes to Existing Features 
+------------------------------------
+*	In UWC-v1.6, Duplicate "cout" prints removed & replaced with UWC logger prints
+*	In UWC-v1.6, Updated readme for RT patch installation steps for ubuntu 20.04
+*	In UWC-v1.6, Removed the PDF version of user guide from https://github.com/open-edge-insights/uwc with Sphinx documentation at  https://github.com/open-edge-insights/uwc-docs
+* In UWC-v1.6, KPI bug of random crashing and zero bugs is fixed
```
Comment (nagdeepgk) [inline source/Pages/page_02.rst:52]: rephrase like this:
KPI bugs on random KPI-app crashing, non-linearity of bad records versus control loop count addressed.

**Review by nagdeepgk — APPROVED** [review]

**Review by nagdeepgk — APPROVED** [review]

**Pull request merged**